ParentWorkspace/SpringModule/src/main/java/com/miniproject/model/Item.java

```diff
@@ -46,10 +46,14 @@
 	@Column(name = "OFFER")
 	private BigDecimal offer;
 	
-	public Item(Integer itemId, Category cId, String item, String brandName, BigDecimal cost, String measure,
+	@Column(name = "IMAGE")
+	private Blob image;
+	
+	@Column(name = "ACTIVE")
+	private boolean active;
+	
+	public Item( Category cId, String item, String brandName, BigDecimal cost, String measure,
 			BigDecimal offer, Blob image) {
```

Comment:
is "active" Boolean variable initialization through constructor was not needed ha ??